src/main/java/models/Card.java

```diff
@@ -8,11 +8,11 @@
  */
 public class Card
 {
-    public final int value;
```

Comment:
We have an A for Ace because Ace can either be 11 or 1. We change it to 11 or 1 based on the total score of the player or dealer at runtime.